src/specfem3D/compute_element.F90

```diff
@@ -482,12 +502,10 @@ subroutine compute_element_tiso(ispec, &
                                           duxdyl_plus_duydxl,duzdxl_plus_duxdzl,duzdyl_plus_duydzl)
 
   ! compute deviatoric strain
-  if (COMPUTE_AND_STORE_STRAIN) then
-    call compute_element_deviatoric_strain(duxdxl,duydyl,duzdzl, &
+  call compute_element_deviatoric_strain(duxdxl,duydyl,duzdzl, &
```

Comment:
You can search for the value init of `COMPUTE_AND_STORE_STRAIN`, for example, https://github.com/geodynamics/specfem3d_globe/search?q=COMPUTE_AND_STORE_STRAIN

Reply:
Awesome, thank  you!